docker/docker-compose.yml

```diff
@@ -5,18 +5,19 @@ services:
       context: ..
       dockerfile: docker/Dockerfile
       args:
-        - TAG=humble-0.21
+        - TAG=foxy-0.21
     environment:
       DISPLAY: $DISPLAY
       XAUTHORITY: $XAUTHORITY
       NVIDIA_DRIVER_CAPABILITIES: all
       ROS_LOG_DIR: /tmp
     container_name: snp
-    image: ghcr.io/ros-industrial-consortium/scan_n_plan_workshop:humble-latest
+    image: ghcr.io/ros-industrial-consortium/scan_n_plan_workshop:foxy-latest
```

Comment:
Let's leave this on Humble, but please change from `humble-latest` to `humble-master` to match the actual name of the image